docker/README.md

```diff
@@ -10,7 +10,7 @@ See [the Docker Compose configurations for our tests databases locally and also
 
 ## Using docker-compose for databases
 
-This is only intended to be run in a development environment where ports 3306 / default for MySQL - 5432 / default for PostgreSQL and 4306 custom port - MariaDB are free and not used.
+This is only intended to be run in a development environment where ports 3306 / default for MySQL - 5432 / default for PostgreSQL and 4306 custom port - MariaDB is free and not used.
```

Comment:
This sentence is a bit hard to understand due to punctuation being off but "are" is correct here since it refers to three ports and not to one.